app/views/metrics/show.html.erb

```diff
@@ -25,18 +25,21 @@
     <%= render "components/related-actions",
       {
         links: [
-                  {
-                    header: t(".navigation.visit_page"),
-                    link_url: "//www.gov.uk#{@performance_data.base_path}",
-                    accessibility_message: t('.navigation.accessibility_message'),
-                    label: @performance_data.title
-                  },
-                  {
-                    header: "Make changes to the page",
-                    link_url: "#",
-                    label: t('.navigation.edit_link', publisher_app: @performance_data.publishing_app)
-                  }
-                ]
+          {
+            header: t(".navigation.visit_page"),
+            link_url: "//www.gov.uk#{@performance_data.base_path}",
+            accessibility_message: t('.navigation.accessibility_message'),
+            label: @performance_data.title
+          },
+          {
+            header: "Make changes to the page",
+            link_url: edit_url_for(
+                        content_id: @performance_data.content_id,
+                        publishing_app: @performance_data.publishing_app
+                      ),
+            label: edit_label_for(publishing_app: @performance_data.publishing_app)
```

Comment:
Can `edit_url_for` and `edit_label_for` be called in the presenter?

Reply:
I guess with the limited Rails experience I have, I'm more used to using helper modules in controllers or views, but looking at the only other helper (MetricsFormatterHelper), it's used in presenters.

I'm still not quite sure how I feel about `SingleContentItemPresenter`, and what it's responsibility is, but I have added methods to call the functions in the helper module now.